thirdeye-persistence/src/main/resources/db/migration/mysql/V1_307_0__add_namespace_configuration_entity_table.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS namespace_configuration_entity
+(
+    id                             bigint(20) PRIMARY KEY AUTO_INCREMENT,
+    namespace                      varchar(200),
+    timezone                       varchar(100) NOT NULL,
+    date_time_pattern              varchar(100) NOT NULL,
+    minimum_onboarding_start_time  bigint(20)   NOT NULL,
+    create_time                    timestamp,
+    version                        int(10),
+    update_time                    timestamp DEFAULT CURRENT_TIMESTAMP,
+    json_val                       text
+    ) ENGINE = InnoDB;
+CREATE INDEX namespace_configuration_id_idx ON namespace_configuration_entity (id);
+CREATE INDEX namespace_configuration_namespace_idx ON namespace_configuration_entity (namespace);
+CREATE INDEX namespace_configuration_create_time_idx ON namespace_configuration_entity (create_time);
```

Comment:
what's the intent of these timestamp indexes ? 
we will only fetch by `namespace` in most cases right? or maybe a full scan for some statistics?

Reply:
not needed for now ig. removing